pages/k8s/operations.md

```diff
@@ -296,3 +294,4 @@ things you may wish to try:
 [logging]: /kubernetes/docs/logging
 [decommission]: /kubernetes/docs/decommissioning
 [get-in-touch]:  /kubernetes/docs/get-in-touch
+[bundle-source]: https://api.jujucharms.com/charmstore/v5/charmed-kubernetes-3/archive/bundle.yaml?channel=stable
```

Comment:
Maybe change this to https://api.jujucharms.com/charmstore/v5/charmed-kubernetes/archive/bundle.yaml?channel=stable instead (remove the revision number) so it always points to latest stable.

Reply:
yes, that's a good plan, thanks!